lib/frozen_record/test_helper.rb

```diff
@@ -0,0 +1,35 @@
+module FrozenRecord
+  module TestHelper
+    def self.load_fixture(model_class, alternate_base_path)
+      @cache ||= {}
+
+      ensure_model_class_is_frozenrecord(model_class)
+
+      @cache[model_class] = {
+        old_base_path: model_class.base_path,
+        old_auto_reloading: model_class.auto_reloading,
+      }
+
+      model_class.auto_reloading = true
+      model_class.base_path = alternate_base_path
+    end
+
+    def self.unload_fixtures
+      @cache.each do |model_class, cached_values|
+        ensure_model_class_is_frozenrecord(model_class)
+
+        model_class.base_path = cached_values[:old_base_path]
```

Comment:
Should we raise an exception if `cached_values` is empty? It would be when we call `unload_fixtures` by itself.

Reply:
💯 Could do.